packages/apps/docs/src/docs/build/authentication/create-transactions.md

```diff
@@ -134,68 +117,69 @@ a Kadena SpireKey wallet might look like for the `n_eef68e581f767dd66c4d4c39ed92
-wallet, the user is redirected to the `returnUrl` you provided. The signed
-transaction and optional parameters are included as URL parameters. If the
-transaction was not successfully signed, the unsigned transaction is returned in
-the URL parameters.
+After signing the transaction, the user is redirected to the `returnUrl` you provided. 
+The signed transaction and optional parameters are returned as parameters using the anchor hashtag (`#`). 
 
-| Parameter      | Type     | Required | Description                                                                                                                                       |
-| -------------- | -------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `transaction`  | string   | Required | A base64 encoded string of the signed or unsigned transaction.                                                                                    |
-| `pendingTxIds` | string[] | Optional | Pending transaction identifiers that enable the application to move forward without waiting for the transaction to be confirmed on the blockchain. |
+If the transaction was not successfully signed, the unsigned transaction is returned.
 
-To verify that a transaction has been successfully signed, you can check the
-`sigs` field in the transaction. If the field has undefined signatures, you
-won't be able to submit the transaction.
+| Parameter | Type | Required | Description |
+| --------- | ---- | -------- | ----------- |
+| `transaction` | string | Required | A base64 encoded string of the signed or unsigned transaction. |
+| `pendingTxIds`| string[] | Optional | Pending transaction identifiers that enable the application to move forward without waiting for the transaction to be confirmed on the blockchain. |
 
-If the transaction has been successfully signed, it is valid for a limited
-period of time as specified in the `ttl` value in the transaction
-`publicMeta` data. During this period of time, the application can call the
-`local/preflight=true` endpoint to check whether the transaction is valid, then
-send it to the blockchain to be executed.
+To verify that a transaction has been successfully signed, you can check the `sigs` field in the transaction. 
+If the field has undefined signatures, you won't be able to submit the transaction.
+
+If the transaction has been successfully signed, it is valid for the period of time set in the `ttl` value in the transaction `publicMeta` data.
+During this period of time, the application can call the `local/preflight=true` endpoint to check whether the transaction is valid, then send it to the blockchain to be executed.
 
 After the transaction is submitted to the blockchain, you can check the transaction status using the request key and chainweb-data API or the Kadena client. 
```

Comment:
I don't think we should encourage dApp developers to use estats to confirm a transaction. The `poll` with a confirmation depth of 4+ should be encouraged.